test/Microsoft.Diagnostics.EventFlow.Inputs.Tests/ActivitySourceInputTests.cs

```diff
@@ -3,7 +3,7 @@
 //  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
 // ------------------------------------------------------------
 
-#if NET5_0 || NETCOREAPP2_1 || NETCOREAPP3_1
+#if NET5_0 || NET5_0 || NETCOREAPP2_1 || NETCOREAPP3_1
```

Comment:
Is this a typo? Maybe `NET6_0`?

Reply:
Yes, that was a typo. Fixed by dropping testing .NET 5.0, which will go out of support in early May 2022